README.rst

```diff
@@ -223,6 +223,29 @@ Each ``InterrogatorFunction`` has the following:
 * ``process_arguments(self, argument_string)``: Instance method that converts the string to arguments (``args`` and ``kwargs``) for the ``aggregator`` expression.
 
 
+Filtering data
+~~~~~~~~~~~~~~
+
+To refine data, filters can be used to reduce the resulting data.
+The current filters are currently supported, but may not work for all data types.
```

Comment:
This is pedantic, but the current filters are supported or These filters are currently supported